pkg/orchestrator/etcd_worker.go

```diff
@@ -35,7 +35,14 @@ import (
 
 const (
 	etcdRequestProgressDuration = 2 * time.Second
-	deletionCounterKey          = "/meta/ticdc-delete-etcd-key-count"
+	// etcdTxnTimeoutDuration represents timeout duration for commit a txn to etcd
+	etcdTxnTimeoutDuration = 30 * time.Second
+	// if no msg comes from a etcd watchCh for etcdWatchChTimeoutDuration long,
+	// the watchCh should be reset
```

Comment:
The meaning and implications of the sentence `the watchCh should be reset` is not clear.